frontend/src/components/MapView/DateSelector/index.tsx

```diff
@@ -51,8 +57,10 @@ const POINTER_ID = 'datePointerSelector';
 const calculateStartAndEndDates = (startDate: Date, selectedTab: string) => {
   const year =
     startDate.getFullYear() -
-    (selectedTab === 'anticipatory_action' && startDate.getMonth() < 3 ? 1 : 0);
-  const startMonth = selectedTab === 'anticipatory_action' ? 3 : 0; // April for anticipatory_action, January otherwise
+    (isAnticipatoryActionLayer(selectedTab) && startDate.getMonth() < 3
```

Comment:
Actually I believe this will only apply for the drought AA, since we don't have the same concept of "windows 1 and 2".

Reply:
Yes, i haven't yet distinguished the content (panel, timeline, etc.), that's what I'm going to do next. that's include logic for window 1 and 2 that I saw in certain parts.

Reply:
The component `AnticipatoryActionDroughtPanel` has been cleaned